CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Flexy Thingworx Connector Changelog
 
+## Version 1.1
+### Features
+- Data is added as Thing properties
+- Updated screenshots and documentation to Thingworx 9
+- Tag quality data is included in data from the Talk2M and direct data paths
+### Bug Fixes
+- Data may not be assigned to the proper data type
+- Data from Talk2M/DataMailbox may not load without the 'useHyphens' option enabled
```

Comment:
The way these are written it sounds like the bugs still exist and you are just documenting them here. Generally you would write something like "Data is assigned to correct data type" or "Fixed bug where data would be assigned to wrong data type".